test/utils/configs.go

```diff
@@ -24,6 +24,10 @@ type TestConfig struct {
 	TursoDbPath    string `koanf:"turso_db_path" validate:"required_if=SkipTursoTests false"`
 }
 
+func newTestConfig() *TestConfig {
+	return &TestConfig{SkipTursoTests: true}
```

Comment:
This just have changed 
```suggestion
	return &TestConfig{SkipSqldTests: true}
```